### DISCUSSION TITLE
Accessing images from flowdeck v2

### DISCUSSION THREAD

**Discussion by EJKT** [bitcraze/discussions#178] [Q&A] [answered]
Hi,

So in theory, correct me if I'm wrong, an optical flow sensor, such as the one on the flow deck, takes images in order to then perform some further analysis. Is it possible to get these images at all, and should it be, is it possible for a live video of the ground below (even if the framerate is low)? Please let me know.

ALSO, on a complete separate note, is it possible to give controls for the motor speeds themselves? I want to have only the back 2 motors spinning at a very low RPM so the CF slides forwards over the ground on a small contraption I made.

Thanks.

**Comment by krichardsson** [comment, marked as answer]
Hi,
Unfortunately there is no API to grab the image from the optical flow camera

Please create a new discussion for for you second question
